app/services/catalog/import_service_plans.rb

```diff
@@ -2,10 +2,9 @@ module Catalog
   class ImportServicePlans
     attr_reader :json
 
-    def initialize(portfolio_item_id)
+    def initialize(portfolio_item_id, options = {:force => false})
```

Comment:
I mentioned an options hash in our discussion, but that was just comparing how passing the name makes it really easy to understand what the flag is meant to do.

I was expecting we'd use keyword parameter, not an options hash:
```ruby
def initialize(portfolio_item_id, force_reset: false)
```

Then the check is:
```ruby
force_reset ? clear_service_plans : check_conflict
```

And the caller would pass:
```ruby
Catalog::ImportServicePlans.new(@plan.portfolio_item_id, force_reset: true).process
```